etc/config/c++.amazon.properties

```diff
@@ -1204,6 +1204,12 @@ libs.etl.url=https://github.com/ETLCPP/etl.git
 libs.etl.versions.trunk.version=trunk
 libs.etl.versions.trunk.path=/opt/compiler-explorer/libs/etl/include:/opt/compiler-explorer/libs/etl/include/etl/profiles
 
+libs.namedtype.name=NamedType
+libs.namedtype.versions=trunk
+libs.namedtype.url=https://github.com/joboccara/NamedType.git
```

Comment:
This URL is meant to direct to the homepage of the library, not the .git repostory itself - So users can go check it out in case of need in their browsers. Deleting the `.git` bit should be enough

Reply:
Yeah. I realized it just after I turned off my machine and went to bed :)
BTW ETL has the same issue. I used it as an example.

Reply:
I saw it and fixed it, thanks for noticing :)